WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -243,11 +243,12 @@ local function ConstructTextEditor(frame)
     end
   end);
 
-  function group.Open(self, data, path, enclose, multipath, reloadOptions)
+  function group.Open(self, data, path, enclose, multipath, reloadOptions, setOnParent)
```

Comment:
setOnGroup?

Reply:
No, setOnGroup implies to me that all of the group would be affected. Which is actually the default behavior.